pyne/partisn.py

```diff
@@ -101,12 +112,16 @@ def write_partisn_input(mesh, hdf5, ngroup, pn, **kwargs):
         true, a linearly spaced grid of starting points is used, with dimension 
         sqrt(num_rays) x sqrt(num_rays). In this case, "num_rays" must be a 
         perfect square.
-    
-    Returns
-    -------
-    PARTISN Input file named by 'input_file' above or the default name.
-        Note: read comments generated in file. Not all variables will be 
-        assigned that are necessary.
+    dg : record array, optional, default = None
+        The output of pyne.dagmc.discretize_geom(). Use this input option if 
+        discretize_geom() has already been run, to avoid duplicating this
+        expensive step.
+    find_per_coarse : int, optional, default = 1
```

Comment:
I found that 1 fine per course works fine, as long as you have at least 7 fine intervals total (or something to that affect). The FNG image I generated above used a fine_per_coarse of 1.

Having anything other than a fine_per_coarse of 1 makes it difficult to read the RTFLUX file, because there is one entry per every fine mesh interval. I figured it couldn't hurt to make it an option anyway.